[kubernetes/distributions] Update note about OOM kill and TCP queue length on GKE (#15247)
DataDog/documentation: content/en/containers/kubernetes/distributions.md

Since Agent 7.26, no specific configuration is required for GKE (whether you run `Docker` or `containerd`).

> **Note**: When using COS (Container Optimized OS), the eBPF-based `OOM Kill` and `TCP Queue Length` checks are supported starting from the version 3.0.1 of the helm chart. There are two settings that need to be set:

Reviewer: ```suggestion
**Note**: When using COS (Container Optimized OS), the eBPF-based `OOM Kill` and `TCP Queue Length` checks are supported starting from the version 3.0.1 of the Helm chart. To enable these checks, configure the following settings:
```
